src/sass/components/_contacts.scss

```diff
@@ -0,0 +1,119 @@
+.contacts{
+    background-color: $pageBg;
+    display:flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items:center;
+    
+
+
+&__head{
+    display:none;
+}
+    
+&__box{ 
+    display:flex;
+    flex-direction: column;
+    gap:20px;
+    max-width:220px;
+    width:100%;
+    justify-content: center;
+   
+ 
+
+    @media(min-width:768px){
+        flex-direction: row;
+        margin-top:119px;
+        margin-bottom:81px;
+    
+    }
+
+}
+    
+
+
+&__container{
+    background-color:$white;
+    box-shadow:0px 8px 30px $contactsBoxShadow;
+    border-radius:24px;
+    
+}
+
+&__content{
+
+    margin:43px 20px;
+}
+
+&__title{
+color:$white;
+border-radius:6px;
+font-weight: 500;
+font-size: 12px;
+line-height: 1,83;
+text-align: center;
+margin-bottom:25px;
+
+    
+}
+&--cafe{
+background-color: $CoffeeBG;
+width:80px;
+}
+&--foodtruck{
+background-color: $lime;
+width:130px;
+}
+
+&__city{
+font-weight: 500;
+font-size: 12px;
+line-height: 1.83;
+letter-spacing: 0.04em;
+margin-bottom:10px;
+}
+&__description{
+font-weight: 500;
+font-size: 12px;
+line-height: 1.5;
+color:$text;
+padding-bottom:25px;
+border-bottom: 1px solid $gray;
+}
+
+&__weekdays{
+font-weight: 500;
+font-size: 12px;
+line-height:1,83;
+padding-top:25px;
+margin-bottom:5px;
+}
+
+&__hours{
+font-weight: 500;
+font-size: 12px;
+line-height: 1,83;
+color:$brand-pink;
+padding-bottom:25px;
+border-bottom: 1px solid $gray;
+margin-bottom:25px;
```

Comment:
brak konsekwencji, wyżej używasz padding, a tutaj margin, choć poprawnie byłoby padding. mimo wszystko użyj to na ul